docs/reference/changelog.rst

```diff
@@ -77,6 +77,16 @@ Changelog
 
   For a complete list of commits, see the `X.Y.Z`_ release on GitHub.
 
+X.Y.Z (2024-MM-DD)
```

Comment:
Asking because this is a new process. Should changes from bumping craft-application in this PR be included now or later?

Reply:
My thought is that if there were new features that affected charmcraft users we'd include them, but the features are kinda under-the-hood so I didn't include them.